pyatv/interface.py

```diff
@@ -434,7 +434,7 @@ def listener(self):
         return self.__listener
 
     @abstractproperty
-    def power_state(self):
+    async def power_state(self):
```

Comment:
This should be left as non-async. Properties  should generally not perform any intensive computations, just return a stored value.

Reply:
The only problem here. If I am changing it to non-async, I am getting the following error:

```
Traceback (most recent call last):
  File "/Volumes/Archive/Development/Github/pyatv/pyatv/__main__.py", line 572, in _run_application
    return await cli_handler(loop)
  File "/Volumes/Archive/Development/Github/pyatv/pyatv/__main__.py", line 390, in cli_handler
    return await _handle_commands(args, config, loop)
  File "/Volumes/Archive/Development/Github/pyatv/pyatv/__main__.py", line 482, in _handle_commands
    ret = await _handle_device_command(args, cmd, atv, loop)
  File "/Volumes/Archive/Development/Github/pyatv/pyatv/__main__.py", line 515, in _handle_device_command
    return (await _exec_command(
  File "/Volumes/Archive/Development/Github/pyatv/pyatv/__main__.py", line 538, in _exec_command
    value = await tmp(*args)
TypeError: object PowerState can't be used in 'await' expression

>>> An error occurred, full stack trace above
```

It seems that cli command cannot be non-async, which makes sense.